robot/rover/config/.bash_aliases

```diff
@@ -16,9 +16,12 @@ alias arm="cd $ROVER/ArmDriverUnit"
 alias wheels="cd $ROVER/MobilePlatform"
 alias rostings="cd $ROSPACKAGES"
 alias mcunode="cd $ROSPACKAGES/src/mcu_control/scripts"
+alias upload="cmake --build $ROVER/build --target "
```

Comment:
@AFknUsername @RealWilliamWells We might wanna add robot/**rover**/config/.bash_aliases to the ~/.bashrc, rn its only the one in base being sourced. So this alias didnt work for me. Im not gonna request that change in this PR since its outta scope